src/Pages/Login.js

```diff
@@ -1,31 +1,53 @@
-import React from 'react';
-import { useHistory } from 'react-router-dom';
+import Axios from "axios";
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
 
 export default function Login() {
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
 
-	const history = useHistory()
-	
-	function login() {
-		localStorage.setItem("isAuthenticated", true)
-		history.replace("/dashboard")
-	}
+  const history = useHistory();
 
+  //this could be changed to an async function
+  //this verification route will work if you run the server from index.js on your local machine
+  //currently the credentials username:Admin password:password will work
+  //a registration route is set up under POST "http://localhost:5000/authentication/register" (also accepts username and pw)
+  function login() {
+    Axios({
+      method: "POST",
+      data: {
+        username: username,
+        password: password,
+      },
+      withCredentials: true,
+      url: "http://localhost:5000/authenticate/login",
+    }).then((res) => {
+      if (res.data === "Successfully Authenticated") {
+        setUsername("");
+        setPassword("");
+        localStorage.setItem("isAuthenticated", true);
+        history.replace("/dashboard");
+      }
+    });
+  }
 
-	return (
-		<>
-			<br />
-			<br /> <br /> <br />
-			<h1>Login Page</h1>
-			<br />
-			<label> user name </label>
-			<input type="text"  />
-			<br />
-			<br />
-			<label> password </label>
-			<input type="text" />
-			<br />
-			<br />
-		<button onClick={login}> Login </button>
-		</>
-	);
+  return (
+    <div className="login">
```

Comment:
and then you could write something like this in the HTML... So that it will be visible only when error is set to true 
<img width="1100" alt="Screen Shot 2020-11-07 at 18 19 57" src="https://user-images.githubusercontent.com/58600732/98447596-4f6dc580-2126-11eb-8541-cac147030802.png">